recipe/meta.yaml

```
@@ -55,10 +55,12 @@ outputs:
         - pytorch {{ pytorch }} {{ torch_proc_type }}*
       run:
         - python
-        - pytorch {{ pytorch }} {{ torch_proc_type }}*
+        - pytorch
```

This should do `>= {{ pytorch }}`, AFAIR.
```suggestion
        - {{ pin_compatible("pytorch", max_pin="x") }}
```

---

It should be `"x.x"` but we also need to pin the GPU aspect.

Does `pin_compatible` do GPU?

---

We have a run exports in the pytorch package that does this.

---

My understanding would be that `pin_compatible` takes care of the lower bound, and the `run_constrained` o the CPU/GPU split